### DISCUSSION TITLE
backfill doc unclear

### DISCUSSION THREAD

**Discussion by aamster** [apache/airflow#28904] [Q&A]
The only thing I found in the docs that describes what "backfill" means is 

> A data filling DAG is created with start_date 2019-11-21, but another user requires the output data from a month ago i.e., 2019-10-21. This process is known as Backfill.

Is this referring to a scenario in which a dag was run on 2019-10-21 to produce data, then the same dag was created again with start date 2019-11-21?

**Comment by potiuk** [comment]
You have a nice explanation here: https://docs.astronomer.io/learn/rerunning-dags

**Reply by potiuk** [reply]
And if you think by reading it that docs in Airflow can be improved - feel free. Airflow is an Open Source Project created by > 2300 contrbutors as of today. Many of those people were users like you who foudn things unclear or wanted to add something and added it.

It's super-easy - just click "suggest a change on this page" and PR will be open. It's as easy to update the docs as it is to ask a question / open discussion here (same UI). I hope that if you find things unclear and get an answer, you will give back by contributing it back for the free software you get. 

We really value such contributions by new users who come to understand airflow  - it's much easier for such users to understand where they are looking for answers and how they could not find it , so they are the best to know where to fix the documentation and how to formulate the docs to make it easier to understand by users like them.

And it's a great way to give back for the - absolutely free - software other community members create in - a lot of them - their free time.  Improving docs that are unclear is the least you can do to give back for all the work they've done.
